### DISCUSSION TITLE
Week 7

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#142] [General]
**Instructional Aspects:**
Some of the instructional aspects that we have encountered have been fun. Others have been a real challenge and it wasn’t without help from youTube and some coding friends that I was able to get through it. My favorite thing that we have done so far is the flexfroggy. It was the first ‘easy’ assignment we had in the class, and it was fun. I liked how they gave you the information you needed before they asked you to do any code. I do feel, especially at the beginning, that I was being thrown in the deep end of coding and asked to tread water and not drawn. I feel that I have grown, but I am still treading water, not quite ready to swim laps. I am a little nervous about the final project if I am being honest. 
**Final Project**
With this in mind, this what I would like to do. A simple webpage with a link to a form to fill out for the school lunch count. The form would be my intermediate level and would be done similar to the registration form that we did a few weeks ago with html and CSS code to make it pretty. Java is still very confusing to me, so I don’t think I will be doing much with it.  I intend to avoid easy mistakes that we learned in the debugging experience this week. I have found myself making stupid errors, such as putting a space or "" where it shouldn't be. This can be very hard to see in a long code. 
What are you thinking for your final project? What code do you have a hard time using?

**Comment by metasean** [comment]
> I do feel, especially at the beginning, that I was being thrown in the deep end of coding and asked to tread water and not drawn. 

Unfortunately, there's so much to learn in such a short timeframe, that there ends up being a "firehose" of information. 😔 


> I feel that I have grown, but I am still treading water, not quite ready to swim laps. I am a little nervous about the final project if I am being honest.

That is an absolutely, positively, normal feeling!  

**_You've got this_**!  

Remember, to ask your classmates questions, ask for help debugging code that you're having challenges with, and answer your classmates' questions.  Believe it or not, reading other people's questions and code will help you level up.  It can be hard at first, but it can really help different concepts "click" into place!  So, give your classmate's an opportunity to help you while leveling up, and take the opportunity to help your classmates while you level up. 😉
